csm/common/errors.py

```diff
@@ -152,11 +191,42 @@ class CsmGatewayTimeout(CsmError):
     a timely response from the upstream server.
     """
 
+    _desc = "Did not receive a timely response from the upstream server"
```

Comment:
Did not receive a timely response from the upstream server --> Unable to get timely response

Reply:
updated